indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -775,15 +796,17 @@ public void childEvent(CuratorFramework client, PathChildrenCacheEvent event) th
                     case CHILD_ADDED:
                     case CHILD_UPDATED:
                       taskId = ZKPaths.getNodeFromPath(event.getData().getPath());
-                      final TaskStatus taskStatus = jsonMapper.readValue(
-                          event.getData().getData(), TaskStatus.class
+                      final TaskAnnouncement announcement = jsonMapper.readValue(
+                          event.getData().getData(), TaskAnnouncement.class
                       );
 
                       log.info(
-                          "Worker[%s] wrote %s status for task: %s",
+                          "Worker[%s] wrote %s status for task [%s] on [%s:%s]",
```

Comment:
can change the message to just be announcement.getTaskLocation so it prints a little nicer